app/src/main/java/org/oppia/app/home/HomeFragmentPresenter.kt

```diff
@@ -55,17 +55,17 @@ class HomeFragmentPresenter @Inject constructor(
     // NB: Both the view model and lifecycle owner must be set in order to correctly bind LiveData elements to
     // data-bound view models.
 
+    internalProfileId = activity.intent.getIntExtra(KEY_NAVIGATION_PROFILE_ID, -1)
```

Comment:
I agree with your point here. For this I have filed an issue: https://github.com/oppia/oppia-android/issues/747

Because this is something which is not a very high priority task. Also, once we finalise what convention we want to follow, it would can be `good-first-issue`.